internal/service/projectipaccesslist/data_source_project_ip_access_list_migration_test.go

```diff
@@ -83,38 +83,37 @@ func TestAccMigrationProjectDSProjectIPAccessList_SettingCIDRBlock(t *testing.T)
 }
 
 func TestAccMigrationProjectDSProjectIPAccessList_SettingAWSSecurityGroup(t *testing.T) {
-	acc.SkipTestExtCred(t)
+	projectName := acctest.RandomWithPrefix("test-acc-migration-project-aws")
```

Comment:
Follow up changes of https://github.com/mongodb/terraform-provider-mongodbatlas/pull/1784 where I forgot to update the migration and data source tests